Automattic/woocommerce-payments
# Add onboarding task incentive badge

Related to https://github.com/Automattic/woocommerce-payments/issues/7036

#### Changes proposed in this Pull Request
Use WC filter `woocommerce_admin_woopayments_onboarding_task_badge` to display server based task badge on WooPayments setup task list.

#### Testing instructions

>[!Note]
> Before server deploy, you'll need to checkout server branch `add/action-incentive-task-info`, and use a local site.

- Create a new site with WC from `update/task-list-item-and-woopayments-task` branch.
- Install WooPayments.
- Payments (1) menu should be visible (action incentive).
- Go to WC Home and complete any WC Home task before `Set up WooPayments` until it becomes the active one.
- `Set up WooPayments` task should have the badge and the new task header, matching the attached screenshot.

### Screenshots

| Before | After |
|-|-|
|<img width="692" alt="Screenshot 2023-09-05 at 10 30 25" src="https://github.com/woocommerce/woocommerce/assets/7670276/7f813fbb-5916-4830-b604-ccd7dd0ada64">|<img width="693" alt="Screenshot 2023-09-05 at 10 24 38" src="https://github.com/woocommerce/woocommerce/assets/7670276/6187674d-a680-4fa2-9021-1910ab04254e">|

-------------------


**Post merge**
